### PR TITLE
Change filename to allow rendering on Puppet Forge.

### PR DESCRIPTION
**Pull request by cdchase**
#### Pull Request (PR) description
Rendering of README on Puppet Forge is broken due to the filename.
This PR changes the filename to match the more common use: README.md

### CODE REVIEW

**Comment by dhoppe** [comment]
It is not enough to rename the file, because the syntax is not Markdown.

**Comment by cdchase** [comment]
Oops. GitHub was doing such a good job rendering it, I didn't even notice! I updated syntax and standardized per PDK template.

Still could use a 'Reference' section and some details.

**Comment by cdchase** [comment]
@dhoppe do you know what that syntax was? It was clearly another type of markup, but I did a little research and could not identify. Wondered if there might be a conversion script around...

File: README.md [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+```
+permission  => [
+  'user::rwx',
+  'group::r-x',
+  'other::r-x',
+  'mask::rwx',
+  'group:webadmin:rwx',
+  'user:apache:rwx',
+],
+    ```
```
Comment (dhoppe) [inline README.md]: Please fix the indentation.

**Comment by dhoppe** [comment]
@cdchase Sorry, I have no idea which kind of syntax that is. Did you check, if this module is compatible to puppet-strings? We already have a rake task, which will generate a REFERENCE.md.

**Comment by cdchase** [comment]
I did not check anything further in this module. puppet-strings is new to me, so thanks for pointing it out. I was only intending on helping to get this visible from Puppet Forge.

**Comment by cdchase** [comment]
Fixes #55

**Pull request merged**